zio-http/src/main/scala/zio/http/codec/TextChunkCodec.scala

```diff
@@ -0,0 +1,79 @@
+package zio.http.codec
+
+import scala.annotation.tailrec
+
+import zio.prelude._
```

Comment:
I am not sure what you use prelude for, but this is currently only a transitive dependency. And I think we should not use it, but rather keep dependencies low.

Reply:
prelude is a part of ZIO ecosystem.